.github/workflows/release.yml

```diff
@@ -57,7 +71,7 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
   publish-to-pypi:
-    needs: [test-package]
+    needs: [release]
```

Comment:
Does this run 'release'? Is it what we want?

Reply:
_publish-to-pypi_ needs _release_ to be successful for running.